### DISCUSSION TITLE
[Github Mobile]: A PR comment containing specific a HTML element (e.g., <button>, <script>) will not show text after HTML element

### DISCUSSION THREAD

**Discussion by meck93** [community/community#41555] [Mobile] [answered]
Is there a way to escape the HTML elements and treat them as text? I'm assuming the comment is sanitized to prevent stored XSS attacks. If it is not possible, no worries. 

![Screenshot_20221212_204555_com.github.android~2.jpg](https://user-images.githubusercontent.com/22457755/207140977-549dc6b1-6987-4307-a27e-218da62622e9.jpg)

![Screenshot_20221212_204606_com.github.android~2.jpg](https://user-images.githubusercontent.com/22457755/207141005-5a9e275a-ce40-45dd-ab64-1596f4ca2bcd.jpg)

**Comment by airtower-luna** [comment, marked as answer]
If you escape the \< it will show as text, and then the rest too because it isn't an HTML element any more. You can simply add a backslash in front of it, or wrap the whole element in backticks to put it into a code section (this is what I'd prefer).

Example:

```
\<button>
`<button>`
```

Renders as:

\<button>
`<button>`

**Reply by meck93** [reply]
Sweet. That's perfect. Thanks a lot.

**Reply by metecanpolat1** [reply]
****

**Comment by metecanpolat1** [comment]
****

**Comment by metecanpolat1** [comment]
Metecanpolat1

**Comment by metecanpolat1** [comment]
Metecanpolat1

**Reply by metecanpolat1** [reply]
@Metecanpolat1

**Reply by BagasAnjasmara** [reply]
Bbbb

**Comment by BagasAnjasmara** [comment]
https://github.com/ALI6VIRUS/GONDRONG

**Comment by RubanChristian** [comment]
> Is there a way to escape the HTML elements and treat them as text? I'm assuming the comment is sanitized to prevent stored XSS attacks. If it is not possible, no worries. 
> 
> ![Screenshot_20221212_204555_com.github.android~2.jpg](https://user-images.githubusercontent.com/22457755/207140977-549dc6b1-6987-4307-a27e-218da62622e9.jpg)
> 
> ![Screenshot_20221212_204606_com.github.android~2.jpg](https://user-images.githubusercontent.com/22457755/207141005-5a9e275a-ce40-45dd-ab64-1596f4ca2bcd.jpg)

Yes Sir
